mapflow/functional/view/data_catalog_view.py

```diff
@@ -29,11 +29,59 @@ def display_mosaics(self, mosaics: list[MosaicReturnSchema]):
     def display_mosaic_info(self, mosaic: MosaicReturnSchema):
         if not mosaic:
             return
-        self.dlg.mosaicInfo.setText("ID: {id} \n"
-                                    "created at {created} \n"
-                                    "tags: {tags}".format(id=mosaic.id,
-                                                          created=mosaic.created_at,
-                                                          tags=", ".join(mosaic.tags)))
+        try:
+            self.dlg.mosaicInfo.setText("ID: {id} \n"
+                                        "created {date} at {h}:{m} \n"
+                                        "tags: {tags}".format(id=mosaic.id,
+                                                              date=mosaic.created_at.date(),
+                                                              h=mosaic.created_at.hour,
+                                                              m=mosaic.created_at.minute,
+                                                              tags=", ".join(mosaic.tags)))
+        except TypeError:
```

Comment:
Why do we use TypeError here? Is it in case tags are None?
It is more concise and obvious to do this before:
```
if mosaic.tags:
    tags_str = ', '.join(mosaic.tags)
else:
    tags_str = ''
```
and use tags_str further